### ISSUE TITLE
🐞 [BUG] - Typos-Footer Sec, alt-Image Element, aria-labels-Form Element, title attribute(Tooltip)-Social Icons Bugs/Issues in Most Files 

### ISSUE THREAD

**Issue by NarendraKoya999** [amupedia2021/Project-Amupedia#579]
### Description

- **In `Footer.js` File, Typo Error in` Folow us on` Sentence and also `needs Tooltip for Social Icons for Screen Readers using title attribute in Social Cards Details`**
- **In Examcard.js File,` alt attribute is not used in Image Elements`**
- **In` Team Folder, Team.js file has Our Team Text is not Properly Highlighting and Teamcard.js File has Missing title attribute for Social Icons to Improve More accessibility for Screen Readers`.**
- **In `Login.js File and index.js-Pages Folder Files`, these files have missing `aria-labels in Form Elements`**

### Current behaviour

- **The above raised issues OR Bugs truly positioned in Specific Files** 

### Expected behaviour

- **I want to rectify the above raised Bugs/Issues in Required Files**
- **If any Other Missing Details similar to above my attributes . Then add that details also**

### Reproduction steps

```bash
1. Go to 'Footer.js, Examcard.js, Team.js, Teamcard.js, Login.js and index.js-Pages Folder'
2. Click on 'those above specific Files'
3. Scroll down to '....'
4. See errors on Those File whatever raised by Me
```

### This Below Screenshot Explains Typo Mistake in Footer Section. I.E) Follow Word has mistake.
![projectamupedia-3](https://github.com/amupedia2021/Project-Amupedia/assets/84512164/bcc19a75-da4f-43ff-93ea-a799a1af1d32)

### Browsers

Chrome

### OS

Windows

### Are you contributing for a program ?

GSSoC2023

### Code of Conduct

- [X] I have read and agree to the Contributing Guidelines of this project.

**Comment by NarendraKoya999** [comment]
Hi @Harshil-Jani . Please Review on this Issue as soon as Possible. Thank You !

**Comment by NarendraKoya999** [comment]
Hi @anmode . Please Review on this Issue ?
